wallet/db.c

```diff
@@ -943,3 +943,31 @@ bool sqlite3_bind_json_escaped(sqlite3_stmt *stmt, int col,
 	int err = sqlite3_bind_text(stmt, col, esc->s, strlen(esc->s), SQLITE_TRANSIENT);
 	return err == SQLITE_OK;
 }
+
+struct amount_msat sqlite3_column_amount_msat(sqlite3_stmt *stmt, int col)
+{
+	struct amount_msat msat;
+
+	msat.millisatoshis = sqlite3_column_int64(stmt, col);
+	return msat;
+}
+
+struct amount_sat sqlite3_column_amount_sat(sqlite3_stmt *stmt, int col)
+{
+	struct amount_sat sat;
+
+	sat.satoshis = sqlite3_column_int64(stmt, col);
```

Comment:
are these `sqlite3_column_int64`s unsigned?

Reply:
i'm pretty sure these are signed ints https://www.sqlite.org/capi3ref.html#sqlite3_int64. storing them as 'blob64' uses the uint_64 type.

Reply:
Yes, but they're perfectly fungible, too.  It'll just look a bit weird in the database.  Perhaps we should do a sweep and change them all?  But that's a separate cleanup; I'll put a FIXME.

Reply:
They are signed, which is why I used to have a check that we don't go over `INT64_MAX` but that seems to have been removed somewhere.

Anyway, storing `u64` and then loading into `u64` seems to be doing the right thing even when > `INT64_MAX` :wink: